File: VB/BandedViewBandDefaultColumn/MyBandedGridColumnCollection.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraGrid.Views.BandedGrid

Namespace BandedViewBandDefaultColumn
	Friend Class MyBandedGridColumnCollection
		Inherits BandedGridColumnCollection
		Public Sub New(ByVal view As DevExpress.XtraGrid.Views.Base.ColumnView)
			MyBase.New(view)
		End Sub

		Protected Overrides Function CreateColumn() As DevExpress.XtraGrid.Columns.GridColumn
			Return New MyBandedGridColumn()
		End Function
	End Class
End Namespace

File: CS/BandedViewBandDefaultColumn/MyBandedGridColumnCollection.cs
using DevExpress.XtraGrid.Views.BandedGrid;

namespace BandedViewBandDefaultColumn
{
	class MyBandedGridColumnCollection : BandedGridColumnCollection
	{
		public MyBandedGridColumnCollection(DevExpress.XtraGrid.Views.Base.ColumnView view)
			: base(view)
		{
		}

		protected override DevExpress.XtraGrid.Columns.GridColumn CreateColumn()
		{
			return new MyBandedGridColumn();
		}
	}
}
